sunpy/database/database.py

```diff
@@ -442,12 +438,8 @@ def _download_and_collect_entries(self, query_result, **kwargs):
                 entry.download_time = datetime.utcnow()
                 yield entry
 
-
     def fetch(self, *query, **kwargs):
-
         """
-        fetch(*query[, path, overwrite, client, progress, methods])
```

Comment:
Isn't this for the doc builds like we have in fido?

Reply:
You shouldn't specify the call sig in the docstring it should be self explanatory.